### PR TITLE
New: Show number of files as tooltip over size on disk

### PR DESCRIPTION
**Pull request by RobinDadswell**
#### Database Migration
NO

#### Description
Adds a box on series info to show the number of files
image

#### Issues Fixed or Closed by this PR

* #4203

### CODE REVIEW

**Comment by RobinDadswell** [comment]
Updated to now look like
image

**Review by markus101 — CHANGES_REQUESTED** [review]
Code looks good, but I find the tooltip blends in too much with the labels below. I'm not sure if we want to inverse the tooltip (dark text or lighter background) or more the tooltip above. @Taloth thoughts?

**Comment by Taloth** [comment]
Afaik Robin is still moving the code to inside Label.js so it's more global.

It blends a bit in the background, but it doesn't need to 'pop'. The original tooltip is inverted. However this is more in line with the other tooltips like Links. I'm ok with a different color scheme.

**Comment by RobinDadswell** [comment]
> Afaik Robin is still moving the code to inside Label.js so it's more global.
> 
> It blends a bit in the background, but it doesn't need to 'pop'. The original tooltip is inverted. However this is more in line with the other tooltips like Links. I'm ok with a different color scheme.

I have paused on that temporarily as work has started to get busy which means I am relaxing heavily in the evenings. 

The colour scheme is the same as the links tooltip as per the comment from Taloth

**Comment by thezoggy** [comment]
is this media files or files in the folder? (which could include a bit more  because of subs/metadata/etc)

**Comment by markus101** [comment]
Media files (same as the size is based on the number of media files).

**Comment by thezoggy** [comment]
> 
> 
> Media files (same as the size is based on the number of media files).

then, just show the number of files with the size to group common context ?

**Comment by RobinDadswell** [comment]
> then, just show the number of files with the size to group common context ? (which it actually does show it already on the size as a tooltip) - that's what I am doing; but it's about converting the logic from tooltip and making it work across all other popups like that.

**Comment by thezoggy** [comment]
quick mockup of what I was talking about:
sonarr

**Comment by Taloth** [comment]
We don't want to add more tags or make existing ones bigger. The point here is to make sure the tooltips work on touch devices, and that's it.

**Comment by austinwbest** [comment]
Still think this should be n/n and not just the amount of files. 50/100 episodes (same concept as the index)

File: frontend/src/Series/Details/SeriesDetails.css [hunk 1]
```diff
@@ -119,6 +119,14 @@
   margin: 5px 10px 5px 0;
 }
 
+.fileCountMessage {
+  font-weight: 300;
+  font-size: 15px;
+  color: #e1e2e3;
```
Comment (markus101) [inline frontend/src/Series/Details/SeriesDetails.css:125]: I think `color: $white` works a bit better, though I think it's better in `Tooltip.css` in the `&.inverse` section:

```
&.inverse {
  background-color: $themeDarkColor;
  box-shadow: 0 5px 10px $popoverShadowInverseColor;
  color: $white;
}
```

Also looks like these are in the wrong order and failing in CI.

**Review by markus101 — CHANGES_REQUESTED** [review]
After playing with the other options I think the styling mostly works, just a change to the text.

I think only showing episode files makes sense in this case given episodes without files don't contribute to the size on disk.

Let's get this landed after the small fixups and we can look at refactoring the rest at another time.

**Comment by RobinDadswell** [comment]
@markus101  ready for another go!

**Pull request merged**